packages/clay-date-picker/src/DayNumber.tsx

```diff
@@ -7,43 +7,81 @@ import {Keys} from '@clayui/shared';
 import classnames from 'classnames';
 import React from 'react';
 
-import {IDay, formatDate, setDate} from './Helpers';
+import {IDay, setDate} from './Helpers';
 
 interface IProps {
 	day: IDay;
-	daySelected: Date;
+	daysSelected: readonly [Date, Date];
 	disabled?: boolean;
+	range?: boolean;
 	onClick: (date: Date) => void;
 }
 
+interface IInterval {
+	start: Date;
+	end: Date;
+}
+
+function isWithinInterval(dirtyDate: Date, interval: IInterval) {
```

Comment:
Do you mean to call the first `date` of the `interval` instead of having `dirtyDate`? I think this will break the functionality because we want to know if `date` is within the range, maybe the name of the function is not helping?

Reply:
I am just saying that the name of the argument to the function should be "date" instead of "dirtyDate". The functionality of the util is fine, just thought the naming of the argument could be simplified.

Using "dirtyDate" doesn't give much value since there is no "cleanDate" to compare it against. It also really doesn't matter what kind of date is passed to the function, it is simply checking if its within a given interval.